src/Conversion/ONNXToKrnl/Math/Reduction.cpp

```diff
@@ -55,6 +61,12 @@ struct ScalarOp<ONNXReduceProdOp> {
   using IOp = MulIOp;
 };
 
+template <>
+struct ScalarOp<ONNXReduceSumV11Op> {
```

Comment:
An alternative naming scheme would be to add the version numbering (with/without) the `V` for versioning at the end: `ONNXReduceSumOpV11`.  I leave it up to you.

Reply:
I thought of that too. But I feel it is better to keep the ONNX....Op format for all ONNX operands.